packages/language-server/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.6.6",
   "author": "Mikhail Gunin <gunka462@gmail.com>",
   "license": "Mozilla Public License 2.0",
-  "main": "dist/index.js",
+  "main": "dist/server.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/moetelo/twiggy.git",
     "directory": "packages/language-server"
   },
-  "bin": "./dist/server.js",
+  "bin": "bin/server.js",
```

Comment:
Npm treats `string` as a valid value for `bin` prop in `package.json`.
https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin
> If you have a single executable, and its name should be the name of the package, then you can just supply it as a string

Although `npm publish --dry-run` complains about string values and prefers object value.
```js
npm WARN publish npm auto-corrected some errors in your package.json when publishing.  Please run "npm pkg fix" to address these errors.
npm WARN publish errors corrected:
npm WARN publish "bin" was converted to an object
```

Should we also run `npm pkg fix` here to convert bin to object?

---

This is kind of out of scope for this pr, but I'm a bit hesitant to do it myself since it can possibly break something for you.

Sass LS does this too:
https://github.com/wkillerud/some-sass/blob/eab00574065b210605013e3733f6ed971389c082/packages/language-server/package.json#L30-L32

Reply:
> Should we also run `npm pkg fix` here to convert bin to object?

We could if that stops npm warning about it.

But looking at [this section](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#bin:~:text=If%20you%20have,the%20node%20executable!), this is exactly what this PR does - sets it to one executable file with bin env direction at the top.

If we want to be completely correct, we should probably follow Sass LS approach.

Reply:
> if that stops npm warning about it

Yep, it does.

Ok then, I will commit that separately into master.